etherhoused-log

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+exec papertrail --min-time "24 hours ago" "program:app/web.1 AND (\"turn_off ${1}\" OR \"turn_on ${1}\")"
```

Comment:
Oh, the only reason I implemented this was so I could get visibility into what my system was doing in order to test the stability effects (or lack thereof) of the client broadcast/multicast changes I made, without having my client permanently hooked up to a PC to monitor it. If you don't want to actually push the feature into production, then I may as well drop that commit.